docs/faq/troubleshooting/not-a-member-of-the-organization.md

```diff
@@ -2,11 +2,33 @@
 
 When you see this message, it means that Codacy can't analyze a commit because the associated email address doesn't belong to any Codacy member or author.
 
-There may be different reasons for this to happen:
+You can check which email address is associated with a commit by hovering the cursor on the name of the contributor on the page for the commit:
```

Comment:
I decided to pull these instructions on how to check the email address of a commit and which email addresses belong to a user account to the top since this can be used to troubleshoot all scenarios described below.